packages/react/impression-area/src/useImpressionRef.tsx

```diff
@@ -11,17 +11,18 @@ type Options = Pick<
 >;
 
 export function useImpressionRef<Element extends HTMLElement>({
-  onImpressionStart: _onImpressionStart,
-  onImpressionEnd: _onImpressionEnd,
+  onImpressionStart: _onImpressionStart = noop,
+  onImpressionEnd: _onImpressionEnd = noop,
   timeThreshold = 0,
   root,
   rootMargin,
   areaThreshold: intersectThreshold = 0,
 }: Options) {
-  const onImpressionStart = usePreservedCallback(_onImpressionStart ?? noop);
-  const onImpressionEnd = usePreservedCallback(_onImpressionEnd ?? noop);
+  const onImpressionStart = usePreservedCallback(_onImpressionStart);
+  const onImpressionEnd = usePreservedCallback(_onImpressionEnd);
```

Comment:
What about specifying a `default parameter` in the form of something like `areaThreshold`?